source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -129,6 +130,13 @@ def _extend_journey(pt_journey, fallback_dp, fallback_extremity):
     pt_journey.distances.bike += aligned_fallback.journeys[0].distances.bike
     pt_journey.distances.car += aligned_fallback.journeys[0].distances.car
 
+    # For start fallback section copy pt_section.origin to last fallback_section.destination
+    # where as for end fallback section copy last pt_section.destination to fallback_section.origin
+    if is_start_fallback:
```

Comment:
It would have been nice to have a test that covers this.

Reply:
done

Reply:
I would prefer unittest